doc/developers/testsuite.html

```diff
@@ -372,13 +356,13 @@
       functionality to gather all profiling files and submit them to cdash
       where we already gather testsuite results
       (see <a href="#submit">below</a>). You can do so by invoking
-      <pre>
-  ctest -DCOVERAGE=YES <...> -S ../tests/run_testsuite.cmake
-      </pre>
+<pre>
+ctest -DCOVERAGE=YES <...> -S ../tests/run_testsuite.cmake
```

Comment:
Must have missed this. Is it legal to write `-DCOVERAGE=ON` instead? This way we always use the same way of spelling the truth value.